core/services/gateway/connector/connector.go

```diff
@@ -157,6 +161,38 @@ func (c *gatewayConnector) SendToGateway(ctx context.Context, gatewayId string,
 	return gateway.conn.Write(ctx, websocket.BinaryMessage, data)
 }
 
+func (c *gatewayConnector) SendToGatewayRoundRobin(ctx context.Context, req api.SendRequest) error {
+	// select a gateway to broadcast in round robin manner
```

Comment:
It's not round-robin if it's random every time. Let's either call it random or actually do round-robin.
Random usually makes things much harder to debug. How about we always use the first available Gateway node for simplicity?

Reply:
sure. will call it `SingleNode` in that case